third_party/flatbuffers/workspace.bzl

```diff
@@ -5,7 +5,7 @@ load("//third_party:repo.bzl", "tf_http_archive")
 def repo():
     tf_http_archive(
         name = "flatbuffers",
-        strip_prefix = "flatbuffers-1.12.0",
+        strip_prefix = "flatbuffers-2.0",
         sha256 = "62f2223fb9181d1d6338451375628975775f7522185266cd5296571ac152bc45",
         urls = [
             "https://storage.googleapis.com/mirror.tensorflow.org/github.com/google/flatbuffers/archive/v1.12.0.tar.gz",
```

Comment:
The download URLs and hash, will also require an update, otherwise the build will fail.